code/modules/client/preferences_savefile.dm

```diff
@@ -223,6 +223,10 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	real_name		= reject_bad_name(real_name)
 	if(!features["mcolor"] || features["mcolor"] == "#000")
 		features["mcolor"] = pick("FFFFFF","7F7F7F", "7FFF7F", "7F7FFF", "FF7F7F", "7FFFFF", "FF7FFF", "FFFF7F")
+	if(!features["tail_human"])
+		features["tail_human"] = "None"
+	if(!features["ears"])
+		features["ears"] = "None"

```

Comment:
If you did that you wouldn't need this, I think mcolor only does this because we don't keep a list of valid mutant race colors to choose, just a few practical examples for body randomizing